src/app/checkout/step-2/step-2.component.js

```diff
@@ -117,6 +117,9 @@ class Step2Controller {
   }
 
   getContinueDisabled () {
+    if (typeof this.isCvvValid !== 'undefined' && !this.isCvvValid) {
+      return true
+    }
```

Comment:
This shouldn't be done all the time, only when we're actually using a credit card.